tests/wallet/test_wallet_utils.py

```diff
@@ -12,19 +11,17 @@ def dummy_coin_state(*, created_height: Optional[int], spent_height: Optional[in
     return CoinState(Coin(bytes(b"0" * 32), bytes(b"0" * 32), 0), spent_height, created_height)
 
 
-def heights(coin_states: List[CoinState]) -> List[Tuple[Optional[int], Optional[int]]]:
+def heights(coin_states: Collection[CoinState]) -> List[Tuple[Optional[int], Optional[int]]]:
```

Comment:
does `Collection` imply that it's ordered? I think this function requires the input to be ordered, and not a `set` for example.

Reply:
I changed it to `Collection` to allow a `Set` which makes testing this line https://github.com/Chia-Network/chia-blockchain/pull/15598/files#diff-0d32919512429d54cb0a91510a71245a1e1b32f461e4358fe1e24a872e41e12aR36 possible. Could have probably also just converted it back to a list before passing into `heights` instead.